openlitespeed/build_bssl.sh.patch

```diff
@@ -0,0 +1,68 @@
+diff --git a/script/build_bssl.sh b/script/build_bssl.sh
+index cb7f229..06b310f 100755
+--- a/script/build_bssl.sh
++++ b/script/build_bssl.sh
+@@ -6,7 +6,11 @@ cd src
+ 
+ 
+ if [ ! -d "boringssl" ]; then
+-    git clone https://github.com/google/boringssl.git
++    # download boringssl and liboqs from oqs
++    echo "Downloading oqs boringssl"
++    git clone --branch master https://github.com/open-quantum-safe/boringssl.git
++
++    git clone --branch main --single-branch --depth 1 https://github.com/open-quantum-safe/liboqs.git
+ fi
+ 
+ if [ -d "go" ]; then
+@@ -17,9 +21,9 @@ echo $PATH
+ fi
+ 
+ cd boringssl
+-git reset --hard
+-git checkout master
+-git pull
++# git reset --hard
```

Comment:
Why is there commented out code? Could this be removed?

Reply:
I had commented out those codes as they were no longer relevant.
These unnecessary comments have been removed.